pom.xml

```diff
@@ -8,7 +8,7 @@
   </parent>
   <groupId>io.quarkiverse.openapi.generator</groupId>
   <artifactId>quarkus-openapi-generator-parent</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.0-quarkus2-SNAPSHOT</version>
```

Comment:
I think we would rather go with a major version bump for the version compatible with Quarkus 3.

@gastaldi WDYT?

Reply:
@ricardozanini ^

Reply:
Agreed. I'll release the last 0.x version today, and for Quarkus 3 support, we do 1.x.

Reply:
@ricardozanini I think they are suggesting 0.x to Quarkus 2 and 1.x to Quarkus 3 and both keep evolving rather than having 1.x.x for Quarkus 3 and 1.x.x-quarkus2 for Quarkus 2.

@gsmet and @gastaldi, can you clarify if my understanding is correct?

Reply:
That's what I meant 😆